### PR TITLE
various basic fixes for behat (esp. related to "resources" -> "ci links")

### PR DESCRIPTION
**Pull request by jasperlieber**
Hi all

I spent some time getting all the tests to pass.  Turned out a lot of the changes were related to the change from "resources" to "ci links".  

Ready for review

## Describe context / purpose for this PR
Get all them tests to pass

## Issue link
none

## Link to MultiDev instance

http://md-{{ ticket number }}-accessmatch.pantheonsite.io

## Checklist for PR author
- [x] I have checked that the PR is ready to be merged
- [x] I have reviewed the DIFF and checked that the changes are as expected
- [x] I have assigned myself or someone else to review the PR

### CODE REVIEW

**Comment by a-pasquale** [comment]
This looks good to me.

I added a Jira ticket to restore the search functionality: https://cyberteamportal.atlassian.net/browse/D8-1452

We can restore the tests dependent on that block after that is complete.

File: tests/behat/features/templates/resources-unauth.feature [hunk 1]
```diff
@@ -7,37 +7,38 @@ Feature: test resource page
...
     Then I should see "Learning"
     Then I should see "login"
-    When I fill in "edit-search--2" with "test"
-    And I wait 4 seconds
-    When I click "test-login-resource"
-    Then I should get a "200" HTTP response
+    # When I fill in "edit-search--2" with "test"
+    # And I wait 4 seconds
+    # When I click "test-login-resource"
+    # Then I should get a "200" HTTP response
```
Comment (a-pasquale) [inline tests/behat/features/templates/resources-unauth.feature:44]: Is this one still not working with the search block restored?

**Review by laheara143 — APPROVED** [review]

File: tests/behat/features/templates/resources-unauth.feature [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@templates
+@wip
```
Comment (laheara143) [inline tests/behat/features/templates/resources-unauth.feature:1]: @templates

**Review by laheara143 — CHANGES_REQUESTED** [review]

File: tests/behat/features/templates/resources-unauth.feature (hunk 2 above)
Comment (jasperlieber) [reply tests/behat/features/templates/resources-unauth.feature:1]: darn it!!! thanks

Comment (jasperlieber) [reply tests/behat/features/templates/resources-unauth.feature]: fixed

File: tests/behat/features/templates/resources-unauth.feature (hunk 1 above)
Comment (jasperlieber) [reply tests/behat/features/templates/resources-unauth.feature]: great catch.  i just restored this.

**Pull request merged**